packages/suite-web-landing/package.json

```diff
@@ -7,28 +7,24 @@
         "type-check:watch": "yarn type-check -- --watch",
         "lint": "eslint . --ext .tsx,.ts",
         "copy-files": "yarn workspace @trezor/suite-data copy-static-files",
-        "dev": "yarn workspace @trezor/suite-data copy-static-files && next",
-        "build:app": "next build && next export -o build",
-        "build": "run-s copy-files build:app"
+        "dev": "rimraf ./build && yarn workspace @trezor/suite-build run dev:landing",
+        "build": "rimraf ./build && yarn workspace @trezor/suite-build run build:landing"
     },
     "dependencies": {
```

Comment:
There are dependencies missing and there are some extras. Both in dependencies and devDependencies

Reply:
Let's discuss if we need `polished` and `react-awesome-reveal` libs.